.circleci/config.yml

```diff
@@ -108,9 +108,6 @@ workflows:
             parameters:
               appraisal: ["graphql-1.9", "graphql-1.10", "graphql-1.11"]
       - release:
-          filters:
-            branches:
-              only: master
```

Comment:
I think we still want this...?

Reply:
I removed this because we don't really need it and it allows us to publish prerelease versions. If we always run it, it just lets semantic-release decide if there should be a release. For the main branch (`master`), it will publish a regular release; for a [prerelease branch](https://semantic-release.gitbook.io/semantic-release/usage/workflow-configuration#pre-release-branches) (e.g. `beta`), it will publish a prerelease and for all others it just won't publish. That's how I was able to publish a beta and test it.